yoti_python_sdk/tests/doc_scan/session/create/test_notification_config.py

```diff
@@ -26,6 +27,7 @@ def test_should_build_correctly(self):
         assert result.auth_token is self.SOME_AUTH_TOKEN
         assert result.endpoint is self.SOME_ENDPOINT
         assert self.SOME_TOPIC in result.topics
+        assert result.auth_type is "BASIC"
```

Comment:
I think we want this to be `None` if it is not set, so we can distinguish between unset, and intentionally set to `BASIC`